PLNPluginSchemaMigration.inc.php

```diff
@@ -48,14 +41,42 @@ public function up() {
 				$table->bigInteger('journal_id');
 				$table->string('uuid', 36)->nullable();
 				$table->bigInteger('status')->default(0)->nullable();
+				$table->string('staging_state')->nullable();
+				$table->string('lockss_state')->nullable();
 				$table->datetime('date_status')->nullable();
 				$table->datetime('date_created');
 				$table->datetime('date_modified')->nullable();
 				$table->string('export_deposit_error', 1000)->nullable();
+				$table->datetime('date_preserved')->nullable();
```

Comment:
It's currently not used, but I'll add it to the interface, mostly to give some relief in case a deposit is in a failed state (the user will be able to see if at least a previous version of it has been preserved).